### ISSUE TITLE
Change Avalanche references to Actian Data Platform

### ISSUE THREAD

**Issue by hab6** [ActianCorp/actian_tableau_connector#46]
Given the rename of Avalanche to Actian Data Platform, we need to update references for the Actian Tableau connector.

Here are a few places that need to be considered for changes.

### Actian Tableau Connector GitHub Repository
```
λ grep -I -r --include "*" -i avalanche .
./actian_jdbc/manifest.xml:<connector-plugin class='actian_jdbc' superclass='jdbc' plugin-version='1.0.4' name='Actian Avalanche JDBC' version='18.1'>
./actian_jdbc/README.md:iijdbc.jar can either be downloaded from [ESD](https://esd.actian.com/product/Avalanche/JDBC/java/Actian_Avalanche_JDBC_Drivers)
./actian_odbc/manifest.xml:<connector-plugin class='actian_odbc' superclass='odbc' plugin-version='1.0.4' name='Actian Avalanche ODBC' version='18.1'>
./actian_odbc/README.md:Install ODBC Driver from [ESD](https://esd.actian.com/product/Avalanche/Client_Runtime)
./actian_odbc/README.md:     * "Options" field takes ODBC Connection String Keywords, see https://docs.actian.com/avalanche/index.html#page/Connectivity/Connection_String_Keywords.htm
./README.md:Tableau connector (aka taco) for Actian Avalanche, Vector, and Ingres.
./README_DEV.md:If your object name case is all lower case (which is the default for Actian Avalanche/Vector/ActianX/Ingres) mixed case option is **not** required.
./tests/datasets/TestV1/actian/README.md:# Hosting TestV1 in Actian Cloud Avalanche local Vector/ActianX/Ingres
```

### Actian Connector Offering on Tableau Exchange

https://exchange.tableau.com/products/167

```
Tableau Connector Plugin to Actian Avalanche, a fully managed hybrid data warehouse that runs on-prem
and across multiple cloud platforms, including AWS, Azure and GCP. Avalanche is designed from the
ground up to deliver high performance at scale across all dimensions – data volume, concurrent user,
and query complexity at a fraction of the cost. This connector offers enhanced SQL support and works
with Actian’s OLAP and OTLP databases.
```
Note: At the time of this update (in Oct 2023), The offering available on Tableau Exchange is Actian ODBC Connector version 0.0.0. In Q4 2023, we expect to replace this offering with the latest Actian JDBC Connector for Tableau.

### Internal References
Confluence [page](https://actian.atlassian.net/wiki/spaces/IE/pages/13795425/Tableau) for the Actian Tableau Connectors

#### _(more to be added)_

**Comment by hab6** [comment]
ODBC files were updated via [53079](http://reviewboard.actian.com/r/53079/)

**Comment by hab6** [comment]
Notes from August 20 when I spoke with Justin Powers about the Avalanche name change to ADP and its impact for the Actian Client build process.

Justin told me:
His team builds the AC and they have fully automated testing.
They do NO development work on the AC. Our team (Gordy, Marius, Vibi) do AC bug fixes.
They have a script called makesaveset that builds the installable package and will need to be updated. He said Manuela should take care of this. Justin will talk with her on Oct 23.
The change will require infrastructure build script updates. Justin will take care of this.
In summary, Justin said he and his team will take care of all the necessary changes to the AC build that are affected by the name change from Avalanche to ADP. I don’t need to follow up with him.

**Comment by hab6** [comment]
On Oct 24, I emailed Actian tech doc contact Brian Marquez about changing the relevant documentation.

Relevant snippet of Brian's response:

It’s official next month, although changes are processing this week.
If you own it, you change it is the best way to look at it. For me, if it’s under ‘[docs.actian.com/avalanche/](http://docs.actian.com/avalanche/)…’ it changes to ‘[docs.actian.com/actiandataplatform](http://docs.actian.com/actiandataplatform)’. I own the doc under this.
Any Github doc would be the owner (your team).

**Comment by hab6** [comment]
To my knowledge, we are done with our responsibility regarding the offering name change. There might be a few other areas (e.g. internal sites) that still need updating. Will deal with additional changes as they are found.
